/bin/zsh: can't open input file: /Users/matthewbixby/Documents/Coding/IntelliJ-daily/daily-cat-fact-committer/run.sh
Successfully committed daily cat fact for 2025-01-06
Successfully committed daily cat fact for 2025-01-07
Successfully committed daily cat fact for 2025-01-08
Successfully committed daily cat fact for 2025-01-09
Successfully committed daily cat fact for 2025-01-10
Successfully committed daily cat fact for 2025-01-11
Successfully committed daily cat fact for 2025-01-12
Successfully committed daily cat fact for 2025-01-13
Successfully committed daily cat fact for 2025-01-14
Successfully committed daily cat fact for 2025-01-15
Successfully committed daily cat fact for 2025-01-16
Successfully committed daily cat fact for 2025-01-17
Successfully committed daily cat fact for 2025-01-18
Successfully committed daily cat fact for 2025-01-19
Successfully committed daily cat fact for 2025-01-20
Successfully committed daily cat fact for 2025-01-21
Successfully committed daily cat fact for 2025-01-22
Successfully committed daily cat fact for 2025-01-23
Successfully committed daily cat fact for 2025-01-24
Successfully committed daily cat fact for 2025-01-25
Successfully committed daily cat fact for 2025-01-26
Successfully committed daily cat fact for 2025-01-27
Successfully committed daily cat fact for 2025-01-28
Successfully committed daily cat fact for 2025-01-29
Successfully committed daily cat fact for 2025-01-30
Successfully committed daily cat fact for 2025-01-31
Successfully committed daily cat fact for 2025-02-01
Successfully committed daily cat fact for 2025-02-02
Successfully committed daily cat fact for 2025-02-03
Successfully committed daily cat fact for 2025-02-04
Successfully committed daily cat fact for 2025-02-05
Successfully committed daily cat fact for 2025-02-06
Successfully committed daily cat fact for 2025-02-07
Successfully committed daily cat fact for 2025-02-08
Successfully committed daily cat fact for 2025-02-09
Successfully committed daily cat fact for 2025-02-10
Successfully committed daily cat fact for 2025-02-11
Successfully committed daily cat fact for 2025-02-12
Successfully committed daily cat fact for 2025-02-13
Successfully committed daily cat fact for 2025-02-14
Error occurred: null
java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:951)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:133)
	at DailyCatFactCommit.fetchDailyCatFact(DailyCatFactCommit.java:63)
	at DailyCatFactCommit.main(DailyCatFactCommit.java:26)
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1028)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:227)
	at java.net.http/jdk.internal.net.http.AsyncSSLConnection.connectAsync(AsyncSSLConnection.java:56)
	at java.net.http/jdk.internal.net.http.Http2Connection.createAsync(Http2Connection.java:468)
	at java.net.http/jdk.internal.net.http.Http2ClientImpl.getConnectionFor(Http2ClientImpl.java:134)
	at java.net.http/jdk.internal.net.http.ExchangeImpl.get(ExchangeImpl.java:94)
	at java.net.http/jdk.internal.net.http.Exchange.establishExchange(Exchange.java:386)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl0(Exchange.java:570)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl(Exchange.java:423)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsync(Exchange.java:415)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:413)
	at java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsyncImpl$7(MultiExchange.java:454)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture.uniHandleStage(CompletableFuture.java:950)
	at java.base/java.util.concurrent.CompletableFuture.handle(CompletableFuture.java:2372)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:444)
	at java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsync0$2(MultiExchange.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	at java.net.http/jdk.internal.net.http.HttpClientImpl$DelegatingExecutor.execute(HttpClientImpl.java:177)
	at java.base/java.util.concurrent.CompletableFuture.completeAsync(CompletableFuture.java:2719)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsync(MultiExchange.java:299)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.sendAsync(HttpClientImpl.java:1047)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:930)
	... 3 more
Caused by: java.nio.channels.UnresolvedAddressException
	at java.base/sun.nio.ch.Net.checkAddress(Net.java:137)
	at java.base/sun.nio.ch.Net.checkAddress(Net.java:145)
	at java.base/sun.nio.ch.SocketChannelImpl.checkRemote(SocketChannelImpl.java:842)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:865)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:210)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:571)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:212)
	... 26 more
Successfully committed daily cat fact for 2025-02-16
Successfully committed daily cat fact for 2025-02-17
Successfully committed daily cat fact for 2025-02-18
Successfully committed daily cat fact for 2025-02-19
Successfully committed daily cat fact for 2025-02-20
Successfully committed daily cat fact for 2025-02-21
Successfully committed daily cat fact for 2025-02-22
Successfully committed daily cat fact for 2025-02-23
Successfully committed daily cat fact for 2025-02-24
Successfully committed daily cat fact for 2025-02-25
Successfully committed daily cat fact for 2025-02-26
Successfully committed daily cat fact for 2025-02-27
Successfully committed daily cat fact for 2025-02-28
Successfully committed daily cat fact for 2025-03-01
Successfully committed daily cat fact for 2025-03-02
Successfully committed daily cat fact for 2025-03-03
Successfully committed daily cat fact for 2025-03-04
Successfully committed daily cat fact for 2025-03-05
Successfully committed daily cat fact for 2025-03-06
Successfully committed daily cat fact for 2025-03-07
Successfully committed daily cat fact for 2025-03-08
Error occurred: Command failed with exit code: 128
java.io.IOException: Command failed with exit code: 128
	at DailyCatFactCommit.executeCommand(DailyCatFactCommit.java:91)
	at DailyCatFactCommit.executeGitCommands(DailyCatFactCommit.java:81)
	at DailyCatFactCommit.main(DailyCatFactCommit.java:46)
Successfully committed daily cat fact for 2025-03-10
Successfully committed daily cat fact for 2025-03-11
Successfully committed daily cat fact for 2025-03-12
Error occurred: null
java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:951)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:133)
	at DailyCatFactCommit.fetchDailyCatFact(DailyCatFactCommit.java:63)
	at DailyCatFactCommit.main(DailyCatFactCommit.java:26)
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1028)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:227)
	at java.net.http/jdk.internal.net.http.AsyncSSLConnection.connectAsync(AsyncSSLConnection.java:56)
	at java.net.http/jdk.internal.net.http.Http2Connection.createAsync(Http2Connection.java:468)
	at java.net.http/jdk.internal.net.http.Http2ClientImpl.getConnectionFor(Http2ClientImpl.java:134)
	at java.net.http/jdk.internal.net.http.ExchangeImpl.get(ExchangeImpl.java:94)
	at java.net.http/jdk.internal.net.http.Exchange.establishExchange(Exchange.java:386)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl0(Exchange.java:570)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl(Exchange.java:423)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsync(Exchange.java:415)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:413)
	at java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsyncImpl$7(MultiExchange.java:454)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture.uniHandleStage(CompletableFuture.java:950)
	at java.base/java.util.concurrent.CompletableFuture.handle(CompletableFuture.java:2372)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:444)
	at java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsync0$2(MultiExchange.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	at java.net.http/jdk.internal.net.http.HttpClientImpl$DelegatingExecutor.execute(HttpClientImpl.java:177)
	at java.base/java.util.concurrent.CompletableFuture.completeAsync(CompletableFuture.java:2719)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsync(MultiExchange.java:299)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.sendAsync(HttpClientImpl.java:1047)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:930)
	... 3 more
Caused by: java.nio.channels.UnresolvedAddressException
	at java.base/sun.nio.ch.Net.checkAddress(Net.java:137)
	at java.base/sun.nio.ch.Net.checkAddress(Net.java:145)
	at java.base/sun.nio.ch.SocketChannelImpl.checkRemote(SocketChannelImpl.java:842)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:865)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:210)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:571)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:212)
	... 26 more
Successfully committed daily cat fact for 2025-03-14
Successfully committed daily cat fact for 2025-03-15
Error occurred: null
java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:951)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:133)
	at DailyCatFactCommit.fetchDailyCatFact(DailyCatFactCommit.java:63)
	at DailyCatFactCommit.main(DailyCatFactCommit.java:26)
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1028)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:227)
	at java.net.http/jdk.internal.net.http.AsyncSSLConnection.connectAsync(AsyncSSLConnection.java:56)
	at java.net.http/jdk.internal.net.http.Http2Connection.createAsync(Http2Connection.java:468)
	at java.net.http/jdk.internal.net.http.Http2ClientImpl.getConnectionFor(Http2ClientImpl.java:134)
	at java.net.http/jdk.internal.net.http.ExchangeImpl.get(ExchangeImpl.java:94)
	at java.net.http/jdk.internal.net.http.Exchange.establishExchange(Exchange.java:386)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl0(Exchange.java:570)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl(Exchange.java:423)
	at java.net.http/jdk.internal.net.http.Exchange.responseAsync(Exchange.java:415)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:413)
	at java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsyncImpl$7(MultiExchange.java:454)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture.uniHandleStage(CompletableFuture.java:950)
	at java.base/java.util.concurrent.CompletableFuture.handle(CompletableFuture.java:2372)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:444)
	at java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsync0$2(MultiExchange.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	at java.net.http/jdk.internal.net.http.HttpClientImpl$DelegatingExecutor.execute(HttpClientImpl.java:177)
	at java.base/java.util.concurrent.CompletableFuture.completeAsync(CompletableFuture.java:2719)
	at java.net.http/jdk.internal.net.http.MultiExchange.responseAsync(MultiExchange.java:299)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.sendAsync(HttpClientImpl.java:1047)
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:930)
	... 3 more
Caused by: java.nio.channels.UnresolvedAddressException
	at java.base/sun.nio.ch.Net.checkAddress(Net.java:137)
	at java.base/sun.nio.ch.Net.checkAddress(Net.java:145)
	at java.base/sun.nio.ch.SocketChannelImpl.checkRemote(SocketChannelImpl.java:842)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:865)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:210)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:571)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:212)
	... 26 more
Successfully committed daily cat fact for 2025-03-17
Successfully committed daily cat fact for 2025-03-18
